src/ast/visitStmt.zig

```diff
@@ -40,6 +40,32 @@ pub fn VisitStmt(
 
         const visitors = struct {
             pub fn s_import(noalias p: *P, noalias stmts: *ListManaged(Stmt), noalias stmt: *Stmt, noalias data: *S.Import) !void {
+                // Handle `import { feature } from "bun:bundle"` - this is a special import
+                // that provides static feature flag checking at bundle time.
+                const import_record = &p.import_records.items[data.import_record_index];
```

Comment:
don't we already have code that tags the import record? is that at parse time? can we move this code to be there instead?